Vialer/ContactsSearchTableViewController.m

```diff
@@ -45,7 +45,7 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     }
 
     // Return the number of rows in the section.
-    NSLog(@"%s: ResultCount -> %ld", __PRETTY_FUNCTION__, (unsigned long)_searchResults.count);
+    //NSLog(@"%s: ResultCount -> %ld", __PRETTY_FUNCTION__, (unsigned long)_searchResults.count);
```

Comment:
Same comment as earlier. Please remove code that you don't need. Do not add commented code that is unnecessary.